### DISCUSSION TITLE
7/17~7/23 글 업로드

### DISCUSSION THREAD

**Discussion by shkisme** [BDD-CLUB/challenge-every-week-posting#127] [1. 개발 관련 글 모음]
[복잡한 조건의 세미나 조회 api 개발하기](https://velog.io/@shkisme/%EB%B3%B5%EC%9E%A1%ED%95%9C-%EC%A1%B0%EA%B1%B4%EC%9D%98-%EC%84%B8%EB%AF%B8%EB%82%98-%EC%A1%B0%ED%9A%8C-api-%EA%B0%9C%EB%B0%9C%ED%95%98%EA%B8%B0)
[Java에서 문제풀이용 스니펫 등록하기](https://velog.io/@shkisme/Java%EC%97%90%EC%84%9C-%EB%AC%B8%EC%A0%9C%ED%92%80%EC%9D%B4%EC%9A%A9-%EC%8A%A4%EB%8B%88%ED%8E%AB-%EB%93%B1%EB%A1%9D%ED%95%98%EA%B8%B0)
[코딩 테스트 언어 정하기](https://velog.io/@shkisme/%EC%BD%94%EB%94%A9-%ED%85%8C%EC%8A%A4%ED%8A%B8-%EC%96%B8%EC%96%B4-%EC%A0%95%ED%95%98%EA%B8%B0)
[React 시작하기](https://velog.io/@shkisme/%EB%A6%AC%EC%95%A1%ED%8A%B8-%EC%8B%9C%EC%9E%91%ED%95%98%EA%B8%B0)
[React 란?](https://velog.io/@shkisme/React-%EB%9E%80)
[React 개발 환경 설정하기](https://velog.io/@shkisme/%EB%A6%AC%EC%95%A1%ED%8A%B8-%EA%B0%9C%EB%B0%9C-%ED%99%98%EA%B2%BD-%EC%84%A4%EC%A0%95%ED%95%98%EA%B8%B0)

한 주 동안 쓴 글이에요! 뭔가 시간이 정신없이 빨리 가네요 벌써 7월 말............

**Comment by llddang** [comment]
ㅋㅋㅋㅋㅋㅋㅋ나도 코딩 테스트 언어 고민인야!
난 결국 아직 C++ 이지만 언젠가 node.js로 넘어가야지..
자바 ㄷㄷ 대단합니다!

옹 Scanner 와 BufferReader 차이 궁금합니다!!
얼른 다루어주길 바랍니당!!

와 저랑은 다르게 아주 열심히 블로그 쓰셨네요!! 대단합니다!!
